### ISSUE TITLE
create_guest_author_from_user_id doesn't migrate Author Headshots

### ISSUE THREAD

**Issue by ckeeney** [Automattic/Co-Authors-Plus#428]
If you create co-authors for all the users using `wp co-authors-plus create-guest-authors`, the co-author objects get created.  However, the co-author feature images should be populated by the WP User author headshots.  This does not seem to be happening.

**Comment by ckeeney** [comment]
The CLI command just makes calls to [`CoAuthors_Guest_Authors->create_guest_author_from_user_id`](https://github.com/Automattic/Co-Authors-Plus/blob/master/php/class-coauthors-guest-authors.php#L1292), and this should probably just be fixed in that function.  I'll update the issue title accordingly.

**Comment by philipjohn** [comment]
Thanks for the report @ckeeney ! Can you enlighten me on one bit;

> WP User author headshots

By headshot, do you mean the Gravatar image?

After creating Guest Authors are you seeing a change in behaviour? E.g. avatars not appearing where they were before running the command?

**Comment by rebeccahum** [comment]
Hmmm. So, in my test case, for the users who had Gravatar images, it seemed to user map it properly to the guest author and display the correct Gravatar under `Users > Guest Authors`.  @ckneeney can you please provide some steps to re-create this issue?

**Comment by ckeeney** [comment]
Sorry, I inherited this WP and this is the WP I have ever had, so I'm not sure if I am doing something non-standard without realizing it.  We don't use Gravatar.  My WP users have a field called "Author Headshot" under "User Extras" at the bottom of the edit user page.

![image](https://user-images.githubusercontent.com/8799010/27616587-9fed3eda-5b65-11e7-9fc3-7414e486618d.png)

I thought headshots for co-author-plus were supposed to go into the feature image of the co-author object.  I wrote my theme to use that field accordingly.  I don't know where I saw this documented, maybe just in a comment somewhere and nowhere official.  

If both of those things aren't how things are intended to work, then feel free to just close this issue.  I manually migrated the headshots to feature images of the co-author objects and everything is working for me now.
